cmd/playlist.go

```diff
@@ -139,6 +156,45 @@ func newListPlaylistTracksCmd() *cobra.Command {
 	return listCmd
 }
 
+func displayTrack(track *spotify.FullTrack) error {
+	// format and display
+	var data [][]interface{}
+	item := []string{
+		string(track.ID),
+		track.Name,
+		track.Album.Name,
+		track.Artists[0].Name,
+		(time.Duration(track.Duration) * time.Millisecond).Truncate(time.Second).String(),
+		strconv.Itoa(track.Popularity),
+		track.Endpoint,
+	}
+	row := make([]interface{}, len(item))
+	for i, d := range item {
+		row[i] = d
+	}
+	data = append(data, row)
+	printSimple([]string{"ID", "Name", "Album", "Artist", "Duration", "Popularity", "Endpoint"}, data)
```

Comment:
this is on me but the following header
```
[]string{"ID", "Name", "Album", "Artist", "Duration", "Popularity", "Endpoint"}
```
should really be a const

Reply:
Make sense. Unfortunately, I don't know enough Go to make that change and googling it is not proving fruitful.